cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricsTableMigration.java

```diff
@@ -167,7 +171,7 @@ public boolean v2MetricsTableExists(HBaseTableUtil tableUtil, int resolution) {
         TableId hBaseTableId =
           tableUtil.createHTableId(new NamespaceId(tableId.getNamespace()), tableId.getTableName());
         boolean doesExist  = tableUtil.tableExists(admin, hBaseTableId);
-        LOG.trace("Table {} exists ? : {}", hBaseTableId.getTableName(), doesExist);
+        LOG.debug("Table {} exists : {}", hBaseTableId.getTableName(), doesExist);
```

Comment:
is this called often? Perhaps you only want to log that if it exists, because that means the migration is not done yet?